src/code/templates/fv-egg-game.js

```diff
@@ -1095,7 +1094,7 @@ export default class FVEggGame extends Component {
                                   selectedChromosomes: fatherSelectedChromosomes };
       return <GenomeView className={parentGenomeClass}  {...uniqueProps} ChromosomeImageClass={FVChromosomeImageView}
                         small={ true } editable={false} userChangeableGenes={ userChangeableGenes } visibleGenes={ visibleGenes }
-                        onAlleleChange={ handleAlleleChange }
+                        onAlleleChange={ handleAlleleChange } isSelectedEmpty={true} chromosomeHeight={122}
```

Comment:
This is the biggest hack of the PR :( In the original version, a selected chromosome was drawn in bold in the parent genome, and in the offspring genome. In the FV version, a selected chromosome is drawn empty in the parent, and filled in the offspring. I hacked in a flag in the parent genome that basically says, treat me as empty if I'm selected.

To fix this correctly, I'm assuming I should make some changes to the way the state gets updated, to correctly reflect the state of the genomes.